decisions/0009-schema-refactor.md

```diff
@@ -0,0 +1,95 @@
+  * `slot:catalyzed_by` will capture any substrate or enzyme that was used to facilitate a chemical conversion process. 
+  * Enzymes that can be used as catalysts are available in the `Class:ProteolyticEnzymeEnum`.
+  * The `slot:catalyzed_by` will be referenced by proteomics workflows.
+
+### Class:DataGeneration
+* `Class:DataGeneration` will be created and take the place of `Class:OmicsProcessing`
+* `Class:OmicsProcessing` will be removed
+  * All reference to `Class:OmicsProcessing` should be eliminated from NMDC schema & Workflows
+  * Typecode `omprc` will still be valid for `Class:DataGeneration`
+* `Class:DataGeneration` will detail the process of inputting a material sample (Biosample or ProcessedSample, some MaterialEntity) into an instrument and generating data (output)
+  * Instrument data will be denoted using `slot:data_category` on `Class:DataObject`.
+  * The output of DataGeneration should always be `instrument_data`
+* `Class:DataGeneration` will have subclasses `Class:NucleotideSequencing` and `Class:MassSpectrometry` 
+  * This allows for the expansion of additional data types as NMDC grows
+* `slot:instrument_used` will have `Range:Class:Instrument`
+  * `Class:Instrument` will have separate enumerated slots for `vendor` and `model`
+  * `slot:id` will be the identifier of the `instrument_used` and `vendor` and `model` will NOT be inlined for `Class:DataGeneration`. Rather, it will be inferred / traced using the instrument's id.
+* Schema support for complex paths of data generation and data processing.
+  * `Class:DataGeneration` can have instances where a single sample has multiple data files that need processed together during WorkflowExecution.  
+  * Relationships between samples and the data objects will be captured using `slot:part_of`, linking the 'child' back to the 'parent'.
+
+### Class:WorkflowExecution & Class:WorkflowChain
+* `Class:WorkflowExecution` will be a subclass of `Class:PlannedProcess`
+* `Class:WorkflowExecution` will replace `Class:WorkflowExecutionActivity` and will be a subclass of `Class:PlannedProcess`
+* To group subclasses of `Class:WorkflowExecution` into implemented steps, `Class:WorkflowChain` was created.
+  * This also provides clear connection back to the DataGeneration output, using `was_informed_by`
+    * `was_informed_by` is being removed from `Class:WorkflowExecution`
+  * Any implementation of `Class:WorkflowExecution` will use `Class:WorkflowChain`. Even when it's a single linked chain (ie implementation of a single subclass of `Class:WorkflowExecution`)
+* `Class:WorkflowExecution` will provide `part_of` with `Range:Class:WorkflowChain`
+  * This replaces 'used' and will no longer be on `Class:WorkflowExecution`
+* Output from `WorkflowExecution` will always have a value on `slot:data_category` of `processed_data` of `Class:DataObject`
```

Comment:
Not being familiar with these terms, I found this bullet point difficult to parse; especially the part after "value".

Reply:
I think my confusion boils down to how "on" is used here, and the separation between the word "value" and the actual value.

Here's what I propose, which I find more clear:
```diff
- * Output from `WorkflowExecution` will always have a value on `slot:data_category` of `processed_data` of `Class:DataObject`.
+ * Output from `WorkflowExecution` will always have the value, `processed_data`, in `slot:data_category` of `Class:DataObject`.
```

Reply:
resolved in recent commit